### PR TITLE
add design history entry for applying again to 3 courses

### PR DESCRIPTION
**Pull request by EmmaFrith**

### CODE REVIEW

File: app/posts/apply-for-teacher-training/2022-03-08-apply-again-three-courses.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
+---
+title: Apply again to 3 courses
+description: We changed our policy so that candidates can apply to 3 courses when applying again.
+date: 2022-02-28
```
Comment (MylesJarvis) [inline app/posts/apply-for-teacher-training/2022-03-08-apply-again-three-courses.md]: We need to update this to the publication date.

File: app/posts/apply-for-teacher-training/2022-03-08-apply-again-three-courses.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+![Course review page where candidates are prompted to add 2 more courses after selecting a course](prompt-to-add-two-more-courses.png "After selecting a course, candidates are prompted to add 2 more courses.")
+
+![Course review page where candidates are prompted to add one more course after selecting 2 courses](prompt-to-add-one-more-course.png "After selecting 2 courses, candidates are prompted to add one more course.")
+
+![Email to candidate when their application is not successful, inviting them to apply again and choose 3 courses.](apply-again-email.png "When an application ends, candidates are invited to apply again and choose 3 courses.")
+
+## Outcome of the policy change so far
+
+On 2 March 2022, 7 working days after launch:
```
Comment (MylesJarvis) [inline app/posts/apply-for-teacher-training/2022-03-08-apply-again-three-courses.md:47]: I can give figures for 9 March 2022:

* 58.3%
* 9%
* 32.7%

Still 2.26 choices.

41 candidates have at least one offer (and many more attending interviews).

**Comment by MylesJarvis** [comment]
Final comment @EmmaFrith @frankieroberto. Do we have a policy of obfuscating magic link IDs, even if it's in QA? I assume they'll expire after a time anyway, so might be a moot point.

File: app/posts/apply-for-teacher-training/2022-03-08-apply-again-three-courses.md (hunk 1 above)
Comment (EmmaFrith) [inline app/posts/apply-for-teacher-training/2022-03-08-apply-again-three-courses.md]:
```suggestion
date: 2022-03-10
```

**Comment by EmmaFrith** [comment]
> Final comment @EmmaFrith @frankieroberto. Do we have a policy of obfuscating magic link IDs, even if it's in QA? I assume they'll expire after a time anyway, so might be a moot point.

Thanks for flagging @MylesJarvis. I'm not sure tbh. I've taken another screenshot without the link, it's not really needed anyway.

File: app/posts/apply-for-teacher-training/2022-03-08-apply-again-three-courses.md (hunk 2 above)
Comment (EmmaFrith) [reply app/posts/apply-for-teacher-training/2022-03-08-apply-again-three-courses.md:47]: thanks @MylesJarvis. If I change this, I'll need to change the rest of the section - so are you happy for me to leave it as is? 

I've specified the time frame in the entry, so as long as we're clear on that, i don't think it matters too much 

we can do another entry later with more up to date figures

**Review by frankieroberto — APPROVED** [review]

**Review by MylesJarvis — APPROVED** [review]
LGTM

**Pull request merged**